docs/diagrams/AcademicClassDiagram.puml

```diff
@@ -0,0 +1,19 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+Package Academic {
+Class Academic
+Class Question
+Class Exam
+Class Attendance
+}
+
+
+Academic *--> "*" Question
+Academic *--> "*" Exam
+Academic *--> "*" Attendance
```

Comment:
Academic is actually more of an aggregation than a composition, Questions, Exams and Attendance are all created before the Academic class is created, and they can exist without it. I think you could just stick to using a normal association.

Reply:
hmmm.. then should `Student` and its field's relationship also be association?